tokio-trace/tokio-trace-core/src/subscriber.rs

```diff
@@ -91,14 +95,19 @@ pub trait Subscriber: 'static {
     /// return `Interest::Never`, as a new subscriber may be added that _is_
     /// interested.
     ///
-    /// **Note**: If a subscriber returns `Interest::never` for a particular
-    /// callsite, it _may_ still see spans and events originating from that
-    /// callsite, if another subscriber expressed interest in it.
+    /// # Note
```

Comment:
_super_ nitpicky, but now that we have two notes, should probably be
```suggestion
    /// # Notes
```